CONTRIBUTING.md

```diff
@@ -378,3 +378,12 @@ google_compute_instance
 #### Add a new provider
 
 For this, please open an issue to describe the provider that you want to add. We will discuss about the best way to help you in the implementation.
+
+
+#### How to update terraform-provider-google
+
+> Only if google provider need to updated
```

Comment:
Do we need the `>`?

Reply:
I wanted to have it as a warning but less scary. But if you prefer we could remove the `>` and keep it as a normal sentence ;)

Reply:
Yes I think is better, `>` normally is used to quote something from somewhere IMO hehe.